.github/release-drafter.yml

```diff
@@ -0,0 +1,26 @@
+name-template: '$RESOLVED_VERSION'
+tag-template: '$RESOLVED_VERSION'
+categories:
+  - title: 'Features'
+    labels:
+      - 'type: feature'
+  - title: 'Chores'
+    label:
+      - 'type: chore'
+  - title: 'Bugs'
+    labels:
+      - 'type: bug'
```

Comment:
```suggestion
      - 'type : bug'
```

Catch me if I'm wrong about the type labels

Reply:
@vnntsu it has to be the same as we configure our label. Seem like ios-template has space after type, let me update this.

![Screenshot 2022-11-28 at 09 54 28](https://user-images.githubusercontent.com/24598204/204181578-a2c54254-a05b-42b7-98c8-d650c2b248b7.png)